Bottom: 9a92a382d9baa90788aadd7d6d7fae7466a40901
Top:    2d9ab2be9f950c8e89819e6bec7f2e4a4d19370d
Author: Ian Kent <raven@themaw.net>
Date:   2013-08-02 18:27:45 +0800

Refresh of autofs-5.0.7-fix-fix-wildcard-multi-map-regression.patch

---

diff --git a/modules/lookup_nisplus.c b/modules/lookup_nisplus.c
index fc31af3..53e4a85 100644
--- a/modules/lookup_nisplus.c
+++ b/modules/lookup_nisplus.c
@@ -622,6 +622,7 @@ int lookup_mount(struct autofs_point *ap, const char *name, int name_len, void *
 			if (rv != CHE_FAIL) {
 				me = cache_lookup_distinct(mc, key);
 				me->status = time(NULL) + ap->negative_timeout;
+			}
 			cache_unlock(mc);
 		}
 		free(mapent);
